install.sh

```diff
@@ -867,65 +889,10 @@ main()
         read ANY
     fi
 
-    . ./src/init/update.sh
-    # Is this an update?
-    if [ "`isUpdate`" = "${TRUE}" -a "x${USER_CLEANINSTALL}" = "x" ]; then
-        echo ""
-        ct="1"
-        while [ $ct = "1" ]; do
-            ct="0"
-            $ECHO " - ${wanttoupdate} ($yes/$no): "
-            if [ "X${USER_UPDATE}" = "X" ]; then
-                read ANY
-            else
-                ANY=$yes
-            fi
-
-            case $ANY in
-                $yes)
-                    update_only="yes"
-                    break;
-                    ;;
-                $no)
-                    break;
-                    ;;
-                  *)
-                    ct="1"
-                    ;;
-            esac
-        done
-
-
-        # Do some of the update steps.
-        if [ "X${update_only}" = "Xyes" ]; then
-            . ./src/init/update.sh
-
-            if [ "`doUpdatecleanup`" = "${FALSE}" ]; then
-                # Disabling update
-                echo ""
-                echo "${unabletoupdate}"
-                sleep 5;
-                update_only=""
-            else
-                # Get update
-                USER_INSTALL_TYPE=`getPreinstalled`
-                USER_DIR=`getPreinstalledDir`
-                USER_DELETE_DIR="$nomatch"
-                USER_OLD_VERSION=`getPreinstalledVersion`
-                USER_OLD_NAME=`getPreinstalledName`
-            fi
-
-            ct="1"
-
-            # We dont need to update the rules on agent installs
-            if [ "X${USER_INSTALL_TYPE}" = "Xagent" ]; then
-                ct="0"
-            fi
-
-        fi
-        echo ""
-    fi
+    # Setting up the installation directory
+    setInstallDir
```

Comment:
Now that we are able to obtain the installation path from Wazuh services, I think we should:
-Return to the installation flow of checking first if Wazuh is installed to generate an upgrade.
-Stop the installation if Wazuh is installed and the user doesn't want to upgrade.
-Only request Wazuh installation path if it isn't already installed.
-Doing so, the check of default path and user path will not be required anymore to determinate if Wazuh is installed.

Reply:
I had been exploring this suggestion. The only problem with this is the fact that the regular expressions to obtain the installation directory from the services files does not work with the old services files.
So, I suggest to keep at least the check of the `ossec-init.conf` file for the old Wazuh versions and, if the file does not exist, it means that we should check the services to see whether we have an installation of the newer versions.

Reply:
You are right. We need to check both, services and ossec-init